DigitalLearningSolutions.Web/Views/Shared/SearchablePage/Configurations/_ThreeQuarterWidthSearchSortAndFilter.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage
+@model BaseSearchablePageViewModel
+
+<partial name="SearchablePage/_Search" model="Model" />
+
+<partial name="SearchablePage/_Sort" model="Model" />

```

Comment:
If we had a way to determine whether search/sort is enabled, we wouldn't need separate partials for all combinations (e.g. _ThreeQuarterWidthSearchAndFilter.cshtml could go) 😢